tests/test_compare_kmer_content.py

```diff
@@ -0,0 +1,134 @@
+"""
+test_compare_kmer_content.py
+
+Tests comparing k-mer content
+"""
+from io import StringIO
+
+import pandas as pd
+import pandas.util.testing as pdt
+import pytest
+
+
+# Fixtures are functions-turned-variables that can be used across multiple
+# tests. conftest.py contains fixtures that can be used by any test file
+@pytest.fixture
+def folder():
+    return "test-folder"
+
+
+@pytest.fixture
+def ksize():
+    return 3
+
+
+@pytest.fixture
+def ksizes():
+    return [2, 3, 4]
+
+
+@pytest.fixture
+def nucleotide_seq1():
+    return "GATTACA"
+
+
+@pytest.fixture
+def nucleotide_seq2():
+    return "GATTTTAAAACA"
+
+
+@pytest.fixture
+def peptide_seq1():
+    return "SASHAFIERCE"
+
+
+@pytest.fixture
+def peptide_seq2():
+    return "YASSSSASHAAAFIERCE"
+
+
+def test_kmerize(nucleotide_seq1, ksize):
+    from khtools.compare_kmer_content import kmerize
```

Comment:
import khtools.compare_kmer_content once in the top and call their functions as compare_kmer_content.kmerize etc and avoid repeating so many lines in every test